next-docs/pages/world/account-delegation.mdx

```diff
@@ -1 +1,74 @@
+
+  We have an example of a delegation control `System` in [the `std-delegations` module](https://github.com/latticexyz/mud/tree/main/packages/world-modules/src/modules/std-delegations).
+
+- `initCallData`, call data for a function that is called on the `delegationControlId` to inform it of the new delegation.
+  This [call data](https://docs.soliditylang.org/en/latest/abi-spec.html) includes both the function to call and the parameters with which it is provided.
+
+## Using a delegation
+
+The delegatee can use [`callFrom`](https://github.com/latticexyz/mud/blob/main/packages/world/src/World.sol#L353-L394).
+This function takes three parameters:
+
+- `delegator`, the address on whose behalf the call is happening.
+- `systemId`, the `System` to call.
+- `callData`, the [call data](https://docs.soliditylang.org/en/latest/abi-spec.html) to send the `System`, which includes both the function to call and the parameters with which it is provided.
+
+## Namespace delegation
+
+The owner of a namespace can use [`registerNamespaceDelegation`](https://github.com/latticexyz/mud/blob/main/packages/world/src/modules/core/implementations/WorldRegistrationSystem.sol#L265-L306) for all access to the namespace.
+This functionality is useful, for example, to implement a trusted forwarder for the namespace.
+This is not a security concern, even though it looks like one, because the namespace owner already has full control over any table or `System` in the name space.
+
+## Gotchas and edge cases
+
+There are several edge cases which could trip people up when it comes to delegation.
+
+### Only one delegation per (delegator, delegatee) pair
+
+The table that specifies the `System` to check for delegation ([`world:UserDelegationControl`](https://github.com/latticexyz/mud/blob/main/packages/world/src/codegen/tables/UserDelegationControl.sol)) has two fields in the key: `delegator` and `delegetee`.
+This means that if a user registers one delegation to let a specific `System` authorize requests for one purpose (for example, requests to a specific namespace), and then another delegation to have a different `System` authorize requests for a differet purpose (maybe a different namespace), within the same `World`, the second registration overwrites the first.
+
+This is not a major problem, it is possible to write a `System` that will authorize requests by looking at the destination namespace and calling a different `System` to authorize requests going to different namespaces, but you should be aware of it.
+
+### Can give `UNLIMITED_DELEGATION` to `address(0)`
+
+There are circumstances that require you to give limited permissions to every user, so sometimes you it makes sense to use `address(0)`.
+MUD also lets you give `UNLIMITED_DELEGATION` to every user, but that does not make sense unless there is nothing in the `World` you care about.
+
+### Order of delegation checks
```

Comment:
My preference would be to remove the `## Gotchas and edge cases` category (commented on the individual paragraphs) and instead only have a `## Order of delegation checks` category (as a h2 instead of h3)